examples/inferencing/running-on-gke.md

```diff
@@ -0,0 +1,185 @@
+   </td>
+  </tr>
+  <tr>
+   <td><code>KSA_NAME</code>
+   </td>
+   <td>N
+   </td>
+   <td>Name of the Kubernetes Service Account configured with access to the given GCS path. By default one will be provisioned as ‘aiinfra-gke-sa’
+   </td>
+   <td><code>my-other-ksa</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>MODEL_NAME</code>
+   </td>
+   <td>N
+   </td>
+   <td>Friendly name for the model, used in constructing the Kubernetes Resource names
+   </td>
+   <td><code>t5-flan</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>INFERENCING_IMAGE_TAG</code>
+   </td>
+   <td>N
+   </td>
+   <td>Image tag for the inference image. Default is ‘release’
+   </td>
+   <td><code>latest</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>USE_FASTER_TRANSFORMER</code>
+   </td>
+   <td>N
+   </td>
+   <td>Boolean to set when the FasterTransformer / Triton path should be enabled.
+<p>
+This controls whether a Conversion job is scheduled, and the inference image that will be deployed.
+   </td>
+   <td><code>true</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>CONVERTED_MODEL_UPLOAD_PATH</code>
+   </td>
+   <td>Y*
+   </td>
+   <td>Only required when USE_FASTER_TRANSFORMER is set.
+<p>
+A GCS path to upload the model after it is converted for FasterTransformer
+   </td>
+   <td><code>gs://pirillo-sct-bucket/converted_t5/1/Model</code>
+   </td>
+  </tr>
+  <tr>
+   <td><code>POD_MEMORY_LIMIT</code>
+   </td>
+   <td>N
+   </td>
+   <td>Sets the memory limit of pods for GKE in <a href="https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-memory">Kubernetes Memory resource format</a>. Defaults to “16Gi”.
+   </td>
+   <td><code>50Gi</code>
+   </td>
+  </tr>
+</table>
+
+
+
+### Running the image
+
+The Cluster Provisioning + Deployment image is available at [gcr.io/llm-containers/gke-provision-deploy](gcr.io/llm-containers/gke-provision-deploy) .
+
+Run the image using the `docker run` command specifying the Environment File with `--env-file`.
+
+After the image finishes provisioning the cluster, the model will be converted (if necessary) and deployed to the cluster. The image will then terminate.
+
+
+### Consuming the Deployed Model
+
+A NodePort service on the cluster is automatically created during deployment. This nodeport allows a user to consume the model on a network that has access to the GKE node.
+
+The IP address of the nodes can be found using the GKE dashboard on Pantheon or using gcloud -> kubectl.
+
+For gcloud, retrieve the kubeconfig file and use kubectl commands to communicate with the cluster.
+
+The cluster name will be `$NAME_PREFIX-gke`.
+
+    $ gcloud containers clusters get-credentials $CLUSTER_NAME --region $REGION --project $PROJECT
+    $ kubectl get nodes –output=wide	# Retrieve the Internal or External IP
+    $ kubectl get svc # Retrieve the Port mapped to 5000 for basic consumption, 8000 for raw consumption
+    $ curl --location 'http://$IP:$PORT/health'
+    200 { "health": "ok" }
+
```

Comment:
Please provide sample output and highlight where the user can copy the IP and port from